src/pages/our_ambassadors.astro

```diff
@@ -228,6 +228,23 @@ import AmbassadorCard from "@components/AmbassadorCard.astro";
         >.
       </AmbassadorCard>
 
+      <AmbassadorCard
+        name="Mahesh Binzer-Panchal"
+        img="mahesh_binzer-panchal.png"
+        country="se"
+        github="mahesh-panchal"
+        mastodon="https://genomic.social/@mbp"
+        linkedin="www.linkedin.com/in/mahesh-binzer-panchal-79a726a2"
+        twitter="arcane_mahesh"
+      >
+        Mahesh is a bioinformatician working for the <a href="https://www.nbis.se/" target="_blank">
```

Comment:
Hi Mahesh.

Could you please shorten your bio? Otherwise, it pushes the social media icons away.
You can check the preview of your PR [here](https://deploy-preview-232--nextflow-staging.netlify.app/our_ambassadors.html).

Reply:
Done.